js/view.js

```diff
@@ -26,12 +26,16 @@ module.exports.highlightArea = function(ids) {
     });
 };
 
-module.exports.printAttractionDetails = function() {
-
+module.exports.printAttractionDetails = function(attraction) {
+    removeAttractionDetails();
+    
+    $("#sidebar").on("click", ".attractionData", function(){
+        $(this).append(detail(attraction));
+    });
 };
 
-module.exports.removeAttractionDetails = function() {
-
+const removeAttractionDetails = function() {
+    $(".attractionDetails").remove();
```

Comment:
Is this supposed to be targeting `.attractionData` in the `.hbs` template?

Reply:
Yes!  Thank you!